### PR TITLE
chore: add Nonso

### PR DESCRIPTION
**Pull request by Arndy345**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1637"></a>

### CODE REVIEW

File: public/data/Nonso.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/nonso_andy",
+      "icon": "twitter"
+    },
+     {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/nonsougbodu/",
+      "icon": "linkedin"
+    },
+   "milestones": []
```
Comment (Panquesito7) [inline public/data/Nonso.json:22]: There's a missing comma in here.
```suggestion
   "milestones": [],
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Arndy345** [comment]
Thanks @Panquesito7

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**